eng/packages/http-client-csharp/generator/Azure.Generator/src/Azure.Generator.csproj

```diff
@@ -11,6 +11,10 @@
     <PackageReference Include="Microsoft.TypeSpec.Generator.ClientModel" />
   </ItemGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\..\plugins\client\Client.Plugin\src\Client.Plugin.csproj" />
```

Comment:
Wondering if we should try to use shared source for the visitors instead of having a dep on the plugin project. We could have a Visitors.csproj under eng/visitors that would host all the visitors. Then we could import the visitors as shared source wherever they are needed.